stdlib/public/Concurrency/Task.swift

```diff
@@ -978,6 +978,31 @@ func _taskCreateNullaryContinuationJob(priority: Int, continuation: Builtin.RawU
 @_silgen_name("swift_task_isCurrentExecutor")
 func _taskIsCurrentExecutor(_ executor: Builtin.Executor) -> Bool
 
+@available(SwiftStdlib 5.9, *)
+@usableFromInline
+@_silgen_name("swift_task_getExecutorRefTypeName")
+func _executorGetTypeNameRaw(_ executor: Builtin.Executor) -> (UnsafePointer<UInt8>, Int)
+
+@available(SwiftStdlib 5.9, *)
+func _executorGetTypeName(_ executor: Builtin.Executor) -> String {
+  let (stringPtr, count) = _executorGetTypeNameRaw(executor)
+  // FIXME: can't use this since it is internal in stdlib and we're in concurrency
+  //  return String._fromUTF8Repairing(
+  //      UnsafeBufferPointer(start: stringPtr, count: count)).0
+  return String(cString: stringPtr)
```

Comment:
**Question**: We can't access the `_fromUTF8Repairing` since it is internal in the stdlib, but that's what we want here tbh... should we perhaps make it visible to the concurrency lib? Can we use @_spi perhaps?

Reply:
Note to self: we can do an SPI for it